src/types/genesys-cloud-media-session.ts

```diff
@@ -305,6 +303,8 @@ export class GenesysCloudMediaSession {
   }
 
   onSessionTerminate (reason?: JingleReasonCondition) {
+    this.state = 'ended';
+
     if (this.peerConnection) {
```

Comment:
Not directly related to this ticket, but from the interface (and a bit of code spelunking), it looks like we should always have a peerConnection. Is there a situation I overlooked where we might not have one?